src/main/java/icons/WaifuMotivatorIcons.kt

```diff
@@ -3,7 +3,15 @@ package icons
 import com.intellij.openapi.util.IconLoader
 
 object WaifuMotivatorIcons {
+    @JvmField
```

Comment:
Turns out that when using Kotlin, we need to annotate it with `@JvmField` so we can properly use it in `plugin.xml` file. see [ref.](https://jetbrains.org/intellij/sdk/docs/reference_guide/work_with_icons_and_images.html#how-to-organize-and-how-to-use-icons)

Reply:
:thinking: Hmm, okay. I see the advantage of this approach. I actually changed this to a Kotlin class to make it a bit easier to work with icons. I don't know if I like the annotations.  However, the current amount of these types of icons is low. Maybe it needs to be revisited if we start adding more icons.